core/templates/components/oppia-angular-root.component.ts

```diff
@@ -1491,7 +1506,83 @@ private injector: Injector
 
     OppiaAngularRootComponent.injector = this.injector;
 
+    // Initialize dynamic meta tags.
+    this.metaTagCustomizationService.addMetaTags([
+      {
+        propertyType: 'name',
```

Comment:
moved it here from `base.ts`.
Unlike `base.ts`, this component is available in both hybrid and angular pages. So, this logic will work for both.